docs/explanation/upgrade.rst

```diff
@@ -0,0 +1,13 @@
+=======
+Upgrade
+=======
+
+This phase is responsible for applying the upgrade plan step by step (See
+:doc:`Upgrade a cloud <../how-to/upgrade-cloud>`). By default steps are applied
+sequentially, but there are steps which can be run in parallel.
```

Comment:
no longer true

Reply:
This is still true, you cannot use `--parallel`, but you can define two parallel steps for single app, and they will be executed automatically in parallel.

Reply:
ok, then... we should reword it. Saying "there are steps which can be run in parallel" seems to imply to me that the user can choose.